config/application.rb

```diff
@@ -15,19 +15,20 @@
 
 module Domus
   class Application < Rails::Application
+    config.app_generators.scaffold_controller :responders_controller
+
+    config.action_controller.action_on_unpermitted_parameters = :raise
     config.load_defaults 5.1
     config.generators.system_tests = nil
-  end
-
-  config.action_controller.action_on_unpermitted_parameters = :raise
 
-  config.generators do |generate|
-    generate.helper false
-    generate.javascript_angine false
-    generate.request_specs false
-    generate.routing_specs false
-    generate.stylesheets false
-    generate.test_framework :rspec
-    generate.view_specs false
+    config.generators do |generate|
+      generate.helper false
```

Comment:
I usually leave this one as the default true, but up to you

Reply:
So probably it would be better, I changed it, thanks! :)